lib/collections/initiatives.js

```diff
@@ -79,15 +85,16 @@ Initiatives.after.update(function initiativesAfterUpdate(userId, doc, fieldNames
     sendFollowersNotification('vote', userId);
   }
 
-  if (modifier.$pull && modifier.$pull.usersVoted) {
+  if(modifier.$pull && modifier.$pull.usersVoted){
     createNotification('remove-vote', userId, doc.createdBy);
   }
 
-  if (modifier.$addToSet && modifier.$addToSet.usersFollowing) {
+  if(modifier.$addToSet && modifier.$addToSet.usersFollowing){
     createNotification('follow', userId, doc.createdBy);
     sendFollowersNotification('follow', userId);
   }
-  if (modifier.$pull && modifier.$pull.usersFollowing) {
+
+  if(modifier.$pull && modifier.$pull.usersFollowing){
```

Comment:
Will fail eslint. requires a space after `if` and space before `{`. In this case it looks as if you deleted teh spaces that existed already?